src/main/java/nl/dtls/fairdatapoint/api/config/ApplicationFilter.java

```diff
@@ -65,6 +71,14 @@ public void doFilterInternal(final HttpServletRequest request,
         ThreadContext.put("ipAddress", request.getRemoteAddr());
         ThreadContext.put("responseStatus", String.valueOf(
                 response.getStatus()));
+        ThreadContext.put("requestMethod", request.getMethod());
+        ThreadContext.put("requestURI", request.getRequestURI());
+        ThreadContext.put("requestProtocol", request.getProtocol());
+        ThreadContext.put("responseStatus", String.valueOf(response.getStatus()));
+        String contentLength = response.getHeader(HttpHeaders.CONTENT_LENGTH);
+        ThreadContext.put("contentSize", contentLength);
+        Marker apiRequest = MarkerFactory.getMarker("API-REQUEST");
+        LOGGER.info(apiRequest, "");
```

Comment:
Markers are created from a factory, they can be made as a static member for this class.